back/spec/models/user_spec.rb

```diff
@@ -34,7 +34,7 @@
       expect(invitee.invitee_invite.invitee.id).to eq invitee.id
     end
 
```

Comment:
I think we need to add some specs here to ensure support for light users. For example: creating a user without a first name, creating a user without slug, and creating a user that did not specify a required field (I think this case is not yet supported in this PR).